### PR TITLE
<MarketingPageLayout/> - new component

### PR DESCRIPTION
**Pull request by sivanharel**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5909/

**Comment by sivanharel** [comment]
@milkyfruit-
I have just finished the development, I made all the changes that were needed according to our conversation.
The only thing I should do is to update the storybook examples. I will do it when you will sent it to me.
Please make a UX review when you have time.

File: src/MarketingPageLayout/test/MarketingPageLayout.spec.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import { marketingPageLayoutPrivateDriverFactory } from './MarketingPageLayout.private.uni.driver';
+
+describe(MarketingPageLayout.displayName, () => {
+  const render = createRendererWithUniDriver(
+    marketingPageLayoutPrivateDriverFactory,
+  );
+
+  afterEach(() => {
+    cleanup();
+  });
```
Comment (EgoziE) [inline src/MarketingPageLayout/test/MarketingPageLayout.spec.js:14]:
```suggestion
  afterEach(cleanup);
```

**Review by EgoziE — APPROVED** [review]

File: src/MarketingPageLayout/MarketingPageLayout.st.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+.root:imageHorizontalPadding:size(medium) .imageContainer {
+  padding-right: value(spacing48);
+}
+
+.root:imageHorizontalPadding:size(large) .imageContainer {
+  padding-right: value(spacing90);
+}
+
+.footerContainer {
+}
```
[the remark this answers is not in the document]
Comment (sivanharel) [reply src/MarketingPageLayout/MarketingPageLayout.st.css:53]: lol.. I use to define padding according to the size prop:
.footerContainer {
}

.root:size(medium) .footerContainer {
  padding: 0 value(spacing48);
}

.root:size(large) .footerContainer {
  padding: 0 value(spacing90);
}

**Review by milkyfruit — CHANGES_REQUESTED** [review]
Hi, @sivan, I went over the PR and wrote down the update story examples and minor fixes.

**Component Fixes**
- [x] missing divider above the footer
- [ ] Size prop should be split into two props – VerticalPadding and HorizontalPadding

**Side Task**
This probably needs a new jira task:
- [ ] `<Text><ul><li>` should have 6px margins between items.
- [ ] `<Text size="small"><ul><li>` should have 9px margins between items. 

### Examples

- [x] **Description**
MarketingPageLayout is a container capable of displaying a brief introduction about a product and its key features. Use it to promote new products that site owners are unaware of.

- [x] **Structure**
Component includes the following sections: a content, an image and a footer. Use content area to add text descriptions, image area to show a preview of promoted product, footer area to emphasize key features or user testimonials.

-  [ ] **Size**
_Let's discuss first if size prop will be split into two, then the text can be adjusted_

Component supports 2 sizes – medium and large. They should be used together with the breakpoints. Size medium should be used for screen sizes that are 1463 by 629 and smaller. Size large should be used for screen sizes that are 1464 by 630 pixels and larger.

medium size:
```
<MarketingPageLayout
  size="medium"
  content={
    <MarketingPageLayoutContent
      size="medium"
      overline="Medium overline"
      title="Medium title"
      content={
        <Text size="small">
          <ul>
            <li>Medium content item one</li>
            <li>Medium content item two</li>
            <li>Medium content item three</li>
          </ul>
        </Text>
      }
      actions={<Button size="medium">Medium Action</Button>}
    />
  }
  image={<Image />}
/>;
```

large size:
```
<MarketingPageLayout
  size="large"
  content={
    <MarketingPageLayoutContent
      size="large"
      overline="Large overline"
      title="Large title"
      content={
        <Text size="medium">
          <ul>
            <li>Large content item one</li>
            <li>Large content item two</li>
            <li>Large content item three</li>
          </ul>
        </Text>
      }
      actions={<Button size="large">Large Action</Button>}
    />
  }
  image={<Image />}
/>;
```

- [x] **Footer with Feature List**
Component lets to insert additional information to the footer. Use FeatureList to display top features that need additional descriptions and visual emphasis.

```
<MarketingPageLayout
  content={
    <MarketingPageLayoutContent
      title="Upgrade Your Site with a Premium Plan"
      content="Wix gives you everything you need for a stunning website. Upgrade your site to Premium and get even more."
      actions={<Button skin="premium" size="large">Upgrade Your Site</Button>}
    />
  }
  image={<Image />}
  footer={
    <FeatureList
      features={[
        {
          id: '0001',
          image: <Image width={60} height={60} />,
          title: 'Remove Wix Ads',
          text:
            "Enjoy a website that's completely your own brand by removing Wix ads.",
        },
        {
          id: '0002',
          image: <Image width={60} height={60} />,
          title: 'Connect a Custom Domain',
          text: 'Get your business found with a custom domain.',
        },
        {
          id: '0003',
          image: <Image width={60} height={60} />,
          title: 'Accept Online Payment',
          text: 'Let your customers and clients pay you online at checkout.',
        },
      ]}
    />
  }
/>;
```

- [x] **Footer With Testimonial List**
Use TestimonialList in the footer to show real user feedback, so site owners can learn how others users see the promoted product.
```
<MarketingPageLayout
  content={
    <MarketingPageLayoutContent
      title="Reach the Right Shoppers on Facebook & Instagram"
      subtitle="Get set up and Wix’s AI will take care of the rest"
      content={
        <Text>
          <ul>
            <li>Show your ad to the people most likely to buy</li>
            <li>Test your ad non-stop to see what’s working </li>
            <li>Optimize your budget to get you the best results </li>
          </ul>
        </Text>
      }
      actions={<Button size="large">Start Now</Button>}
    />
  }
  image={<Image />}
  footer={
    <TestimonialList
      testimonials={[
        {
          id: '0001',
          avatar: <Avatar name="Guy in glasses" size="size60" />,
          text: 'I love it! It helped me to increase the sales.',
          authorName: 'John Smith',
        },
        {
          id: '0002',
          avatar: <Avatar name="Person with a hat" size="size60" />,
          text: 'Amazing! It is simple to use, I had no struggle to setup.',
          authorName: 'Templeton Peck',
        },
        {
          id: '0003',
          avatar: <Avatar name="Smiling lady" size="size60" />,
          text: 'A perfect tool for creating ad campaigns that sell.',
          authorName: 'Bosco B.A.',
        },
      ]}
    />
  }
/>;
```

**Comment by sivanharel** [comment]
@milkyfruit - Hi Domas, I made all the changes that you asked besides the splitting of the size prop.
It is not clear what did you mean when writing: "Size prop should be split into two props – VerticalPadding and HorizontalPadding".

As you said at our last meeting the padding is added to the content and to the image sections but can be removed only from the image so we decided to change the naming of the props: "sidePadding" ==> "ImageHorizontalPadding" , "verticalPadding" ==> "ImageVerticalPadding".

In addition, we said that the padding values are changed according to the size prop:
medium size ==> the vertical padding (top and bottom) is 30px, the horizontal padding (right and left) is 48px
large size ==> the vertical padding (top and bottom) is 48px, the horizontal padding (right and left) is 90px

Regarding the side tasks can you please open Jira tasks for them?

The Partners team is waiting for this component so let's discuss about "the size prop" and push to merge it.

**Comment by milkyfruit** [comment]
@sivanharel in order not to block Partners Team, let's launch without `size` prop at all for now.
Paddings should be controlled separately:
- top and bottom: medium: 30, large: 48 
- left and right: medium: 48, large: 90

**Comment by sivanharel** [comment]
@milkyfruit - ok fine I can remove the size prop but if I will do it how will the padding be controlled? 
We have to define if it's medium or large in order to know how much space to declare. So we need a prop for it...

**Review by milkyfruit — APPROVED** [review]
@sivanharel the changes look good, I think it can be merged.

File: stories/symbolsComponentsMapping/symbols.js [hunk 3]
```diff
@@ -29,6 +29,7 @@ export const foundationSymbols = {
  */
 export const layoutSymbols = {
   pageLayout: '2.1 Page Layout',
+  marketingPageLayout: '2.1f Marketing Page',
```
Comment (shlomitc) [inline stories/symbolsComponentsMapping/symbols.js:32]: is the `f` in "2.1f" intentional?

File: src/MarketingPageLayout/MarketingPageLayout.st.css [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+.root:horizontalSize(medium) .contentContainer {
+  padding-right: value(spacing48);;
+  padding-left: value(spacing48);;
+}
+
+.root:horizontalSize(large) .contentContainer {
+  padding-right: value(spacing90);;
+  padding-left: value(spacing90);;
+}
```
Comment (shlomitc) [inline src/MarketingPageLayout/MarketingPageLayout.st.css:25]: extra `;`

**Review by shlomitc — APPROVED** [review]
great, really minor stuff

File: stories/symbolsComponentsMapping/symbols.js (hunk 3 above)
Comment (sivanharel) [reply stories/symbolsComponentsMapping/symbols.js:32]: As I understood yes (it's also written in the spec) but I will verify it with Domas again.

Comment (milkyfruit) [reply stories/symbolsComponentsMapping/symbols.js:32]: yes, in UX Story it's indexed as part of 2.1 Page story as one of the layouts:
image

**Pull request merged**